lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -113,6 +120,51 @@
   </div>
 %endif
 
+%if settings.FEATURES.get('ENABLE_INSTRUCTOR_BACKGROUND_TASKS') and section_data['access']['instructor']:
```

Comment:
I would argue that these checks should be separate because it defines access for this particular feature. In the case we have to modify access for this feature, we should not have to touch code that belongs to another feature.